lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -80,7 +80,7 @@ def handle_event_file(event_file)
         logger.debug("[foreman_ansible] - parsing event file #{event_file}")
         begin
           event = JSON.parse(File.read(event_file))
-          if (hostname = event.dig('event_data', 'host'))
+          if hostname_for_event(event)
```

Comment:
I guess this hould be:
```suggestion
          if (hostname = hostname_for_event(event))
```